packages/create-next-app/templates/index.ts

```diff
@@ -103,7 +103,7 @@ export const installTemplate = async ({
       stats: false,
       // We don't want to modify compiler options in [ts/js]config.json
       // and none of the files in the .git folder
-      ignore: ["tsconfig.json", "jsconfig.json", ".git/**/*"],
+      ignore: ["tsconfig.json", "jsconfig.json", ".git/**/*", "**/fonts/**"],
```

Comment:
probably can't hurt (and might fix the failing turbopack tests? see https://github.com/vercel/next.js/pull/69806#issuecomment-2334911568)
```suggestion
      ignore: ["tsconfig.json", "jsconfig.json", ".git/**/*", "**/fonts/**", "**/favicon.ico"],
```